### PR TITLE
Clarified use of name fields in OAuth instructions.

### PR DESCRIPTION
**Pull request by sarahmaddox**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/377)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-377--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-gke/#create-oauth-client-credentials

File: content/docs/started/getting-started-gke.md [hunk 1]
```diff
@@ -42,35 +42,48 @@ You do not need a running GKE cluster. A cluster will be created when you execut
...
    
      ![consent-screen](/docs/images/consent-screen.png)
-1. On the [Credentials](https://console.cloud.google.com/apis/credentials) screen:
+1. On the [credentials tab](https://console.cloud.google.com/apis/credentials):
    * Click **Create credentials**, and then click **OAuth client ID**.
    * Under **Application type**, select **Web application**.
-   * In the **Name** box enter any name.
+   * In the **Name** box enter any name for your OAuth client ID. This is *not*
+     the name of your application nor the name of your Kubeflow deployment. It's
+     just a way to help you identify the OAuth ID.
```
Comment (yebrahim) [inline content/docs/started/getting-started-gke.md]: nit: OAuth **client** ID

Comment (sarahmaddox) [reply content/docs/started/getting-started-gke.md]: Done.

**Comment by sarahmaddox** [comment]
Thanks @yebrahim PTAL.

**Comment by yebrahim** [comment]
/lgtm
/approve

Thanks Sarah.

**Pull request merged**